### DISCUSSION TITLE
Custom landscape problems

### DISCUSSION THREAD

**Discussion by Richielisa** [Stellarium/stellarium#2496] [Q&A]
Hi,

I have created a custom landscape and have added this successfully to Stellarium but no matter what number I put in to the rotatez it does not alter the orientation of the landscape to the compass setting in Stellarium. Can anyone help please?

thanks

Richie

**Comment by gzotti** [comment]
Check the actual name described in the User Guide, chapter 7:

Polygonal landscape:
polygonal_angle_rotatez

Spherical/fisheye landscape:
angle_rotatez

old_style_landscape:
decor_angle_rotatez

**Reply by Richielisa** [reply]
Thanks for replying. Unfortunately, Im thick when it comes to technology. I read through chapter 7 and felt like I was reading a different language.  Doh!
